Header header
robot_msgs/MechanismState mech_state
SensorSample[] samples